app/models/user.rb

```diff
@@ -307,6 +307,20 @@ def cached_followed_tag_names
         id: Follow.where(
           follower_id: id,
           followable_type: "ActsAsTaggableOn::Tag",
+          points: 1..,
```

Comment:
This ensures that `cached_followed_tag_names` really only includes followed tags.